core/domain/takeout_service_test.py

```diff
@@ -0,0 +1,491 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for core.domain.takeout_service."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+from constants import constants
+from core.domain import exp_domain
+from core.domain import exp_services
+from core.domain import feedback_services
+from core.domain import rights_manager
+from core.domain import takeout_service
+from core.platform import models
+from core.tests import test_utils
+import feconf
+import utils
+
+(
+    user_models, collection_models, exploration_models, story_models,
+    feedback_models, suggestion_models,
+    email_models) = models.Registry.import_models([
+        models.NAMES.user, models.NAMES.collection, models.NAMES.exploration,
+        models.NAMES.story, models.NAMES.feedback, models.NAMES.suggestion,
+        models.NAMES.email])
+
+
+class TakeoutServiceTests(test_utils.GenericTestBase):
```

Comment:
Make it TakeoutServiceUnitTests (FIne with what you have but I see `...UnitTest` pattern in other test files.)

Reply:
done